engines/sci/engine/script_patches.cpp

```diff
@@ -1515,6 +1515,134 @@ static const uint16 gk1Day5MoselyVevePointsPatch[] = {
 	PATCH_END
 };
 
+// When entering the police station (room 230) sGabeEnters sets ego speed
+//  to 4 for the door animation but fails to restore it to the game speed
+//  by calling GKEgo:normalize. This leaves ego at 75% speed until doing
+//  something that does call normalize.
+//
+// We fix this by calling GKEgo:normalize after Gabriel finishes walking
+//  through the door in sGabeEnters:changeState(5). This requires overwriting
+//  the instructions in state 4 which set GKEgo:ignoreActors to 0 but that's
+//  okay because normalize does that.
+//
+// There are two versions of this patch due to two significantly different
+//  versions of this script. The first is in english pc floppy prior to
+//  Sierra's 1.0b patch and the second is in cd and localized floppies. The
+//  script was restructured and the compiler used different sized instructions.
+//
+// Applies to: All PC Floppy and CD versions. TODO: Test Mac, should apply
+// Responsible method: sGabeEnters:changeState
+// Fixes bug #10780
+static const uint16 gk1PoliceEgoSpeedFixV1Signature[] = {
+	0x31, 0x1f,                     // bnt 1f [ state 5 ]
+	SIG_ADDTOOFFSET(+19),
+	0x38, SIG_UINT16(0x0143),       // pushi  [ ignoreActors ]
```

Comment:
There is also SIG_SELECTOR16(), please use that too.

If you use a selector for patching that is not yet available, please add it to the 2 tables at the start of this .cpp file.